.htaccess

```diff
@@ -15,6 +15,9 @@
     # Deny access to framework directories
     RewriteRule ^(app|bootstrap|config|database|resources|storage|tests|vendor)/ - [R=404,L,NC]
 
+    # Deny access to node_modules directories
+    RewriteRule ^node_modules/ - [R=404,L,NC]
```

Comment:
Why another rule and not using the existing rule?
Also, can you fix this into the sample nginx config as well?